src/controllers/update/update.sign.out.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { NextFunction, Request, Response } from "express";
+
+import { createAndLogErrorRequest, logger } from "../../utils/logger";
+import * as config from "../../config";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.UPDATE_SIGN_OUT_URL}`);
+
+    return res.render(config.UPDATE_SIGN_OUT_PAGE, {
+      previousPage: `${config.UPDATE_AN_OVERSEAS_ENTITY_URL}${req.query["page"]}`,
+      url: config.UPDATE_AN_OVERSEAS_ENTITY_URL
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `POST ${config.UPDATE_SIGN_OUT_PAGE}`);
```

Comment:
use logger.debugRequest(req, `${req.method} ${req.route.path}`); for log message